### PR TITLE
Replacing deprecated add success method

### LINKED ISSUE #29667: [Issue] Replacing deprecated add success method
This issue is automatically created based on existing pull request: magento/magento2#28402: Replacing deprecated add success method

---------

### Description (*)

Replaced deprecated method "addSuccess()" with "addSuccessMessage()".

### PR DESCRIPTION
**Pull request by gokulra92**
### Description (*)

Replaced deprecated method "addSuccess()" with "addSuccessMessage()".

### Resolved issues:
1. [x] resolves magento/magento2#29667: Replacing deprecated add success method

### CODE REVIEW

**Comment by gokulra92** [comment]
@magento run all tests

**Comment by edenduong** [comment]
@magento run all tests

**Comment by sidolov** [comment]
@magento create issue

File: app/code/Magento/Integration/Controller/Adminhtml/Integration/Delete.php [hunk 1]
```diff
@@ -44,7 +44,7 @@ public function execute()
                         $this->_oauthService->deleteConsumer($integrationData[Info::DATA_CONSUMER_ID]);
                     }
                     $this->_registry->register(self::REGISTRY_KEY_CURRENT_INTEGRATION, $integrationData);
-                    $this->messageManager->addSuccess(
+                    $this->messageManager->addSuccessMessage(
                         __(
                             "The integration '%1' has been deleted.",
                             $this->escaper->escapeHtml($integrationData[Info::DATA_NAME])
```
Comment (VladimirZaets) [inline app/code/Magento/Integration/Controller/Adminhtml/Integration/Delete.php:50]: addSuccessMessage works a little bit differently way then addSuccess method. By default, it escapes all HTML. So, please check that message is correct and also you can remove  additional escape `$this->escaper->escapeHtml`

File: app/code/Magento/Integration/Controller/Adminhtml/Integration/Save.php [hunk 2]
```diff
@@ -164,7 +164,7 @@ private function processData($integrationData)
                 $integration = $this->_integrationService->update($integrationData);
             }
             if (!$this->getRequest()->isXmlHttpRequest()) {
-                $this->messageManager->addSuccess(
+                $this->messageManager->addSuccessMessage(
                     __(
                         'The integration \'%1\' has been saved.',
```
Comment (VladimirZaets) [inline app/code/Magento/Integration/Controller/Adminhtml/Integration/Save.php:169]: The same here

File: app/code/Magento/Integration/Controller/Adminhtml/Integration/TokensDialog.php [hunk 3]
```diff
@@ -28,7 +28,7 @@ protected function _setActivationSuccessMsg($isReauthorize, $integrationName)
             "The integration '%1' has been activated.",
             $integrationName
         );
-        $this->messageManager->addSuccess($successMsg);
+        $this->messageManager->addSuccessMessage($successMsg);
```
Comment (VladimirZaets) [inline app/code/Magento/Integration/Controller/Adminhtml/Integration/TokensDialog.php:31]: the same here

**Review by VladimirZaets — CHANGES_REQUESTED** [review]
Hi @gokulra92. Thanks for collaboration. Please take a look at my review comments

**Comment by VladimirZaets** [comment]
@magento run all tests

**Comment by engcom-Charlie** [comment]
Hi gokulra92, can you please look through the requested changes https://github.com/magento/magento2/pull/28402#pullrequestreview-470357277 and check failed tests?
Thank you.

**Comment by engcom-Charlie** [comment]
Hi, @gokulra92 there is a lot of changed files that caused too many failures. I would recommend separating these changes to separate PRs. For example, you can replace deprecated methods per module.
What do you think?

**Comment by sidolov** [comment]
@gokulra92  I am closing this PR now due to inactivity.
Please reopen and update if you wish to continue.
Thank you for the collaboration!

**Pull request closed without merging**